first/style.css

```diff
@@ -0,0 +1,240 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap'); /* font-family: 'Rubik', sans-serif */
+
+body {
+    font-family: 'Rubik', sans-serif;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.header {
+    position: relative;
+}
+
+.header img {
+    width: 100%;
+    position: relative;
+}
+
+.navBar {
+    z-index: 2;
+    position: sticky;
+    top: 5px;
+    background-color: white;
+    width: 100%;
+    height: 50px;
+    box-shadow: 0 1px 10px #000000;
+    border-radius: 2px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.navBar ol {
+    width: 20%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    justify-content: space-between;
+    margin-top: 12px;
+    margin-right: 20px;
+    list-style: none;
+}
+
+.navBar ol li {
+    font-size: large;
+}
+
+a {
+    text-decoration: none;
+    color: #000000;
+}
+
+.navBar p {
+    margin-top: 12px;
+    margin-left: 20px;
+}
+
+.navBar p span {
+    font-weight: bolder;
+}
+
+.textPhoto {
+    font-size: 40px;
+    position: absolute;
+    top: 48%;
+    left: 40%;
+}
+
+.textPhoto span {
+    color: white;
+    background-color: #000000;
+    padding: 5px 15px 5px 10px;
+    opacity: 0.75;
+}
+
+.projectsHeder {
+    width: 95%;
+    margin: 1.3rem auto;
+}
```

Comment:
- fix class name 
- why not full width?
- use px for margin (do not use another unit)